GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeOrchestratorTests.cs

```diff
@@ -62,41 +62,21 @@ public void UpgradeSequence()
         }
 
         [TestCase]
-        public void InvalidUpgradeRing()
+        public override void NoneLocalRing()

```

Comment:
Why do you have an override that does nothing but call the base method? What is the goal here?

Reply:
As discussed, review and changes to existing TestCases can be done in a separate PR.